amplify.yml

```
@@ -24,8 +24,7 @@ frontend:
         - tar -xvzf git-lfs.tar.gz
         - mv git-lfs-3.6.1/git-lfs ./bin/
         - chmod +x ./bin/git-lfs
-        # Install git-lfs and pull down the files
-        - export PATH=$PATH:$(pwd)/bin && git-lfs install
+        # Pull down the files
         - export PATH=$PATH:$(pwd)/bin && git-lfs pull
```

Should we simplify this now that we're only calling git-lfs once?

```suggestion
        - ./bin/git-lfs pull
```

---

Unfortunately, that will cause an error in Amplify, since Git tries to call `git-lfs` after the pull has finished.